Packet++/src/TcpReassembly.cpp

```diff
@@ -227,6 +227,11 @@ TcpReassembly::ReassemblyStatus TcpReassembly::reassemblePacket(Packet& tcpData)
 	if (tcpReassemblyData->twoSides[sideIndex].gotFinOrRst)
 	{
 		PCPP_LOG_DEBUG("Got a packet after FIN or RST were already seen on this side (" << static_cast<int>(sideIndex) << "). Ignoring this packet");
+		if (!tcpReassemblyData->twoSides[1 - sideIndex].gotFinOrRst && isRst)
```

Comment:
not very familiar with this piece of code, could you explain why it's `1 - sideIndex` here?

Reply:
1-sizeIndex is mean other side tcp stream，the sideIndex's value maybe 0 or 1.